app.js

```diff
@@ -74,14 +71,19 @@ app.get("/emails/:id", async (req, res, next) => {
 app.get("/emails/:id/download", async (req, res, next) => {
   try {
     const messages = await fetchMessages();
-    const email = messages[req.params.id];
+    const message = messages[req.params.id];
 
-    const parsed = await simpleParser(email.RawData);
+    if (!message.RawData) {
+      res.status(400).send("Can't download emails without RawData!")
+      return;
+    }
```

Comment:
could you please cover this by tests?

Reply:
Are you running tests with `jest`?
I'm getting `SyntaxError: Cannot use import statement outside a module`, and I've tried many things already.

Reply:
done